ui/tests/integration/components/pki/pki-sign-intermediate-form-test.js

```diff
@@ -14,9 +14,7 @@ import { setupMirage } from 'ember-cli-mirage/test-support';
 const selectors = {
   form: '[data-test-sign-intermediate-form]',
   csrInput: '[data-test-input="csr"]',
-  toggleSigningOptions: '[data-test-toggle-group="Signing options"]',
-  toggleSANOptions: '[data-test-toggle-group="Subject Alternative Name (SAN) Options"]',
-  toggleAdditionalFields: '[data-test-toggle-group="Additional subject fields"]',
+  toggleGroup: (group) => `[data-test-toggle-group="${group}"]`,
```

Comment:
Note: there is a [toggleGroup](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L47) in the general-selectors.ts file.

Reply:
Thanks! Since we don't import general selectors here, so I was just following the existing pattern